WebService/ComparisonEngineUI/ComparisonEngineUI.Android/ComparisonEngineUI.Android.csproj

```diff
@@ -32,6 +32,12 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <AndroidLinkMode>None</AndroidLinkMode>
+    <AotAssemblies>false</AotAssemblies>
+    <EnableLLVM>false</EnableLLVM>
+    <AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
+    <BundleAssemblies>false</BundleAssemblies>
```

Comment:
Does this part need to repeat in line 51-54?

Reply:
Weird, it got autogenerated this way